### PR TITLE
docs(contributing): add additional step in the feature flags section

### PR DESCRIPTION
**Pull request by daschaa**
[Follow-Up to this Comment.](https://github.com/aws/aws-cdk/pull/20521#discussion_r916749609)
> Can you also add this info [here](https://github.com/aws/aws-cdk/blob/68f09b7c6f8e6c1ddf13fdd4e116d12333d24c46/packages/%40aws-cdk/cx-api/README.md?plain=1#L14)? We haven't found the best way to document these yet so until then I figure that will have to do.

In the contributing documentation for the feature flags the part for adding an entry in the feature flags README was added in this PR.

----

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md)

### Adding new Unconventional Dependencies:

* [ ] This PR adds new unconventional dependencies following the process described [here](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md/#adding-new-unconventional-dependencies)

### New Features

* [ ] Have you added the new feature to an [integration test](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md)?
	* [ ] Did you use `yarn integ` to deploy the infrastructure and generate the snapshot (i.e. `yarn integ` without `--dry-run`)?

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -958,8 +958,9 @@ The pattern is simple:
    [cx-api/lib/features.ts](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/cx-api/lib/features.ts).
    This map is inserted to generated `cdk.json` files for new projects created
    through `cdk init`.
-4. In your tests, use the `testFutureBehavior` and `testLegacyBehavior` [jest helper methods] to test the enabled and disabled behavior.
-5. In your PR title (which goes into CHANGELOG), add a `(under feature flag)` suffix. e.g:
+4. Add an entry for your feature flag in the [README](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/cx-api/README.md) file.
```
Comment (corymhall) [inline CONTRIBUTING.md:961]: cc @TheRealAmazonKendra. I've started adding things here for the time being, but not sure what you think.

Comment (TheRealAmazonKendra) [reply CONTRIBUTING.md:961]: No objections.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -958,8 +958,9 @@ The pattern is simple:
...
+5. In your tests, use the `testFutureBehavior` and `testLegacyBehavior` [jest helper methods] to test the enabled and disabled behavior.
```
Comment (TheRealAmazonKendra) [inline CONTRIBUTING.md:962]: So, now that we're on v2, these tests no longer work the way we expect them to. I realize your change isn't to this line, except to update the numbering, but could you edit this line to just say to test with and without the feature flags enabled?

Comment (daschaa) [reply CONTRIBUTING.md:962]: Yes, of course! Thanks for the hint. I struggled with this as well when I implemented a feature flag.

Comment (TheRealAmazonKendra) [reply CONTRIBUTING.md:962]: I'm working on a rewrite for feature flags and the testing utilities but I don't yet have the design proposal done so it's going to be a minute. In the meantime I don't want to keep misleading people. Thanks for making this change for us!

**Review by TheRealAmazonKendra — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (daschaa) [reply CONTRIBUTING.md]: I was very pleased to do that. :)

**Pull request merged**